pkgs/applications/networking/cluster/timoni/update.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=./. -i bash -p curl gnused nix
```

Comment:
```suggestion
#!nix-shell -I nixpkgs=./. -i bash -p curl gnused nix jq
```
Needs `jq` to run (used in line 10)

Reply:
Thanks a lot. Forced pushed fix.